include/battle.h

```diff
@@ -289,7 +289,8 @@ struct AI_SavedBattleMon
     u16 ability;
     u16 moves[MAX_MON_MOVES];
     u16 heldItem;
-    u16 species;
+    u16 species:15;
+    u16 saved:1;
```

Comment:
Was this change made to prevent a different battler from overriding data?

Reply:
It's future proofing so you can't overwrite the saved battler data by saving it twice. There wasn't a current issue but it was set up to easily cause that to occur (it happened in a custom frontier facility AI I had written)